src/AppBundle/Controller/BundleController.php

```diff
@@ -252,6 +262,42 @@ public function getOrderedSearchBundlesAction($searchText, $page = 1, $order = n
     }
 
     /**
+     * @param string $order
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function renderOrderForm($order)
+    {
+        $sortingForm = $this->formFactory->create(OrderType::class, [
+            'order' => $order,
+        ]);
+
+        return $this->templating->renderResponse(
+            '@ezdesign/form/order.html.twig',
```

Comment:
Same feedback:  I think `'@ezdesign/form/bundle_sort_order.html.twig'` is better.